rc/servers.kak

```diff
@@ -511,6 +511,21 @@ hook -group lsp-filetype-toml global BufSetOption filetype=toml %{
     }
 }
 
+hook -group lsp-filetype-typst global BufSetOption filetype=typst %{
+    set-option buffer lsp_servers %exp{
+        [tinymist]
+        root_globs = [".git", ".hg"]
+        args = ["lsp"]
+        settings_section = "tinymist"
+        [tinymist.settings.tinymist]
```

Comment:
I think for consistency with others this should be `[tinymist.settings._]` (and `settings_section = "_"`), unless the server really requests the `tinymist` section via `workspace/configuration`, which I doubt because the vscode config file has no such section but only keys like `"tinymist.foo": ...`.

Put another way, the `tinymist` language server probably has no idea that the section is called `tinymist`.
The VS Code plugin likely muddies the waters by collecting all `"tinymist.foo"` keys into one object for `initializeOptions` and `didChangeConfiguration`.
Other editors need to model that ugliness; I think to reduce confusion we should make it explicit that we don't need the `tinymist` name for that.